### ISSUE TITLE
[Feature request] Add radar map from openweathermap.org

### ISSUE THREAD

**Issue by Sitavi** [WangDaYeeeeee/GeometricWeather#214]
Il would be very nice to have the radar map from openweathermap.org, I think they have an api for that, especially for FOSS.
https://openweathermap.org/weathermap?basemap=map&cities=true&layer=radar

I see two ways to include it:

-As a new card (like sun & moon for example).
The map would be zoned on the city and its surroundings and we could interact with it directly in the component (move, zoom). Maybe by clicking, the map could open wide. And there would be buttons like on the other components to choose between Global Precipitation, Pressure, Temperature, Wind speed, & Clouds.

-We could also access the map in a tab independent of the main page (as does the radaweather app on f-droid), it could be zoned on our city of residence or on our location if we have activated the option . There would also be the possibility to choose between Global Precipitation, Pressure, Temperature, Wind speed, & Clouds.

It would be a good way to make the app even more complete by allowing you to have a visual look on a map, especially for precipitation :)

**Comment by zomboto** [comment]
I agree with Sitavi. Would greatly appreciate this feature.  It's the only thing lacking in an otherwise great app.  I have searched long and hard for a great open source weather app to replace the popular ones on play store. With radar maps I would never have to go back to closed source.

**Comment by jafuuu** [comment]
A must need. Adding this would put this app in a competitive (in functionality) state with most closed source app.

**Comment by bestouff** [comment]
FWIW [RadarWeather](https://github.com/woheller69/weather) offers what you want (weather and precipitation radar) but it's far less nice than GeometricWeather. I'd love to have the precipitations radar in the latter.

**Comment by Nitrousoxide** [comment]
I can also get behind adding a radar map.  I find it incredibly useful to be able to see if my area is going to get hit with rain when there are spotty rainstorms in the area to just be able to go to a radar map, watch it cycle through a couple hours of animation quickly to see the general paths of the rainy patches and know if any look to be headed my way.

**Comment by BradB58** [comment]
I agree, that the only feature I have found lacking is a radar map. Hopefully this will be implemented soon and then I will no longer need to have two weather apps on my phone.
